include/executor/executor.h

```diff
@@ -125,6 +125,7 @@ class HostFuncHandler {
 /// Executor flow control class.
 class Executor {
 public:
+  /// TODO add another argument to this for coredump builder to initialise it.
```

Comment:
We don't need this anymore since there is no need to initialize the core dump builder.